src/plugins/intel_npu/tools/single-image-test/main.cpp

```diff
@@ -259,6 +260,7 @@ void parseCommandLine(int argc, char* argv[]) {
     std::cout << "    Mean_values [channel1,channel2,channel3]  " << FLAGS_mean_values << std::endl;
     std::cout << "    Scale_values [channel1,channel2,channel3] " << FLAGS_scale_values << std::endl;
     std::cout << "    Skip checking output layers:              " << FLAGS_skip_output_layers << std::endl;
+    std::cout << "    Clamp u8 output:                          " << FLAGS_clamp_u8 << std::endl;
```

Comment:
We should have a more explicit name for the option, if it's only for output:
```suggestion
    std::cout << "    Clamp U8 outputs:                          " << FLAGS_clamp_u8_outputs << std::endl;
```

Reply:
Fixed in [384f801](https://github.com/openvinotoolkit/openvino/pull/29592/commits/384f801021c8bed64467cc8cee19d48354af27ac)